src/azure-cli/azure/cli/command_modules/role/_msgrpah/_graph_client.py

```diff
@@ -57,7 +57,10 @@ def _send(self, method, url, param=None, body=None, api_version=V1_0):
                 r = send_raw_request(self._cli_ctx, method, url, resource=self._resource, uri_parameters=param,
                                      body=body)
             except HTTPError as ex:
-                raise GraphError(ex.response.json()['error']['message'], ex.response) from ex
+                err = ex.response.json()['error']
+                code = err['code']
+                message = err['message']
+                raise GraphError(f"({code}) {message}", ex.response) from ex
```

Comment:
Will `code` and `message` always exist? Should we use `err.get('Code', '')`?

Reply:
Wait. `code` and `message` are guaranteed to exist: https://learn.microsoft.com/en-us/graph/errors#error-resource-type

![image](https://github.com/user-attachments/assets/129a1bda-26ba-4389-b74d-99610f366741)